README.md

```diff
@@ -2,7 +2,7 @@
 
 Unofficial Swift Package for communicating with Pl@ntNet API.
 
-To get access to the API you need an API key. See [https://my.plantnet.org/account/doc] for API docs and setting up access in general.
+To get access to the API you need an API key. See [my.plantnet.org/account/doc](https://my.plantnet.org/account/settings) for API docs and setting up access in general.
```

Comment:
Thanks for the update!

However, I just checked, the right link should be: `https://my.plantnet.org/doc` (i.e. without `/account/`)

```suggestion
To get access to the API you need an API key. See [my.plantnet.org/doc](https://my.plantnet.org/doc) for API docs and setting up access in general.
```

Reply:
If you just want the API Key, using /settings also work though.

Reply:
Yeah but the comment in the Readme was just meant to link to the official docs so that people know how to start. 

Actually, also the other link is not working.

Can you remove `account` in both URLs, then everything is fine 🙂